rollbar/__init__.py

```diff
@@ -865,11 +867,14 @@ def _add_locals_data(data, exc_info):
             # Fill in all of the named args
             for named_arg in named_args:
                 if named_arg in local_vars:
-                    args.append(local_vars[named_arg])
+                    args[named_arg] = local_vars[named_arg]
 
             # Add any varargs
             if arginfo.varargs is not None:
-                args.extend(local_vars[arginfo.varargs])
+                if SETTINGS['locals']['scrub_varargs']:
+                    args[arginfo.varargs] = ['*' * randint(3, 20) for i in range(len(local_vars[arginfo.varargs]))]
```

Comment:
Alright, this may be a bit more complicated than I anticipated. I'm going to make a PR to this branch containing the Scrub code.